File: Data Structures & Algorithms/anagram-groups/submission-0.js
class Solution {
    /**
     * @param {string[]} strs
     * @return {string[][]}
     */
    groupAnagrams(strs) {
        const result = {};

        for (let word of strs)
        {
            const countKey = new Array(26).fill(0);

            for (let character of word)
            {
                countKey[character.charCodeAt(0) - 'a'.charCodeAt(0)]++;
            }

            const key = countKey.join(',');

            if (!result[key])
            {
                result[key] = [];
            }

            result[key].push(word);
        }

        return Object.values(result);
    }
}

File: Data Structures & Algorithms/count-paths/submission-0.cs
public class Solution {
    public int UniquePaths(int m, int n)
    {
        int[] noOfPathsPerRow = new int[n];
        Array.Fill(noOfPathsPerRow, 1);

        for (var i = m - 2; i >= 0; i--)
        {
            for (var j = n - 2; j >= 0; j--)
            {
                noOfPathsPerRow[j] = noOfPathsPerRow[j] + noOfPathsPerRow[j + 1];
            }
        }

        return noOfPathsPerRow[0];
    }
}
